app/assets/javascripts/discourse/app/controllers/full-page-search.js

```diff
@@ -202,6 +245,18 @@ export default Controller.extend({
     return page === PAGE_LIMIT;
   },
 
+  @discourseComputed("search_type")
+  usingDefaultSearchType(searchType) {
+    return searchType === 0;
```

Comment:
Could we refer to the constant by variable name rather than hard coding it? You might need to change `SearchTypes` a bit.